qmail-smtpd.c

```diff
@@ -33,6 +34,9 @@ GEN_SAFE_TIMEOUTWRITE(safewrite,timeout,fd,_exit(1))
 
 char ssoutbuf[512];
 substdio ssout = SUBSTDIO_FDBUF(safewrite,1,ssoutbuf,sizeof(ssoutbuf));
+#ifdef SMTPUTF8
+int  smtputf8 = 0, flagutf8 = 0;
+#endif
```

Comment:
Let's just always define these, in a separate `eai_smtpd.c` if that helps readability.

Reply:
And every one on it's own line. Since they are global variables they are also automatically initialized to 0. Adding an explicit initializer is actually less efficient.

Reply:
I will do away with flagutf8.